.github/workflows/publish.yml

```diff
@@ -0,0 +1,37 @@
+name: Publish Package to nuget
+
+on:
+  release:
+    types: [create]
+    
+jobs:
+  build:
+
+    env:
+      BUILD_CONFIG: 'Release'
+      SOLUTION: 'intacct-sdk-net.sln'
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Setup NuGet
+      uses: NuGet/setup-nuget@v1.0.5
+
+    - name: Setup .NET
+      uses: actions/setup-dotnet@v2
+      with:
+        dotnet-version: 6.x
+        
+    - name: Restore dependencies
+      run: dotnet restore
+
+    - name: Build
+      run: dotnet build $SOLUTION --configuration $BUILD_CONFIG --no-restore
+      
```

Comment:
please add `-name: Test` as well

Reply:
no need to do that for this operation as the testing portion will happen on the merge to main branch. This is solely for publishing what we know works...... if that's improper thinking, let me know.